.github/workflows/build-test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [16.x, 18.x, 20.x]
+        node-version: [18.x, 20.x]
```

Comment:
Are we avoiding adding `22.x` here due to CI fails? https://github.com/MetaMask/eth-json-rpc-middleware/actions/runs/9993141696/job/27619854892?pr=317

Apparently that issue is fixed as of v22.5.1 https://github.com/nodejs/node/pull/53935 (@cryptodev-2s), so it should be safe to add v22 if we can specify a minimum version in this field.